.orchestra/ci/ci.sh

```diff
@@ -186,6 +186,13 @@ if test "$PUSH_CHANGES" = 1; then
             fi
         done
 
+        # Switch orchestra config from next-* to *
+        # Needed because fix-binary-archive-symlinks
+        # uses the current orchestra config branch name
+        ORCHESTRA_CONFIG_BRANCH="$(git -C "$ORCHESTRA_ROOT" rev-parse --abbrev-ref HEAD)"
+        ORCHESTRA_CONFIG_BRANCH=${ORCHESTRA_CONFIG_BRANCH#next-}
+        git -C "$ORCHESTRA_ROOT" switch "$ORCHESTRA_CONFIG_BRANCH"
```

Comment:
Wait, is this necessary at all?
The loop above should have done this already, isn't it?

Reply:
You are right, `git checkout -b` creates **and switches** to the new branch, I missed it